src/gluonts/torch/modules/distribution_output.py

```diff
@@ -141,11 +141,16 @@ def distribution(
         if loc is None and scale is None:
             return distr
         else:
+            scale = 1.0 if scale is None else scale
             transform = AffineTransform(
                 loc=0.0 if loc is None else loc,
-                scale=1.0 if scale is None else scale,
+                scale=scale,
             )
-            return TransformedDistribution(distr, [transform])
+            transformed_distr = TransformedDistribution(distr, [transform])
+            TransformedDistribution.variance = property(
```

Comment:
This seems a bit weird…

Reply:
you are right! I've fixed it I hope! please kindly review whenever you get time!